packages/server/lib/utils/auth.ts

```diff
@@ -0,0 +1,29 @@
+import type { LogContext } from '@nangohq/logs';
+import type { ApiError, IntegrationConfig } from '@nangohq/types';
+
+import type { Response } from 'express';
+import type { RequestLocals } from './express';
+
+export async function checkIfIntegrationIsAllowed({
```

Comment:
I know I have proposed this name (sorry) but `isIntegrationAllowed` would be even shorter and better